src/main/java/com/rarchives/ripme/utils/Base64.java

```diff
@@ -51,25 +57,34 @@ public static String encode(byte[] buf) {
      * @return the byte array (not null)
      */
     public static byte[] decode(String s) {
-        int delta = s.endsWith("==") ? 2 : s.endsWith("=") ? 1 : 0;
+        int delta = s.endsWith("==") ? 2 : endsWithEqualSymbol(s);
```

Comment:
I don't think this improves readability. It looks like a boolean but it returns an int. I'd prefer if you wrap this expression in parentheses to improve readability.